### ISSUE TITLE
the default release isn't really 1 millisecond

### ISSUE THREAD

**Issue by squinkylabs** [sfzformat/sfzformat.github.io#57]
the spec says default for ampeg_release is .001. But that sounds terrible. Is seems like a lot of SFZ don't specify ampeg_release. But they still sound ok because sforzando seems to default to something more like 20 milliseconds.

I know this "spec vs. sforzando" is an ancient issue, but this is a case where following the spec makes your player sound bad.

**Comment by kinwie** [comment]
It's 0.001. It have been checked here :
https://github.com/sfztools/sfizz/issues/392

**Comment by squinkylabs** [comment]
yes, that's what the spec says, but people author for the sforzando player, not the spec. The default for sforzando is what people author for. So, for example, I made a player, and made it follow the spec of .001 (one millisecond). So now it sounds bad because a lot of SFZ files don't have an ampeg_release override. I will update my player to be more like sforzando and not follow the spec.

I did notice (earlier) that the spec does mention this issue in "volume" where it says the spec is a max of +6 db but many players go much higher. That was useful information. Just suggesting similar info about ampeg_release would be useful to have in the spec.

**Comment by kinwie** [comment]
The original sfz v1 specs said it was 0 for default ampeg_release. I remember we had it corrected to 0.001 from the test result in rgc sfz player, DropZone and sforzando, also in sfizz. You can see the wav image result in the link above. There's a time ruler shows it is 0.001 s.

**Comment by squinkylabs** [comment]
Ok. I don't believe you, though, because I know 1 ms is way to fast to prevent clicks. I measured sforz RTAS running in VCV host last night:

![image](https://user-images.githubusercontent.com/16314838/115321944-ce394680-a139-11eb-9dd6-474486b4a7c1.png)

But that's ok. We don't have to agree.

**Comment by kinwie** [comment]
It's really default to 0.001 :)

**Comment by DSmolken** [comment]
We should try to get to the bottom of this. Can you both maybe check the following:

ampeg_release not set
ampeg_release=0.001
ampeg_release=0.02

That would let us know for certain whether the "not set" looks like 0.001 or like a larger value.

**Comment by jpcima** [comment]
Sforzando's release default looks like 30ms
These two are matching on scope
`<region> sample=*sine amp_veltrack=0 ampeg_release=0.03`
`<region> sample=*sine amp_veltrack=0`

**Comment by squinkylabs** [comment]
Yes, I will try to do a full comparison like @DSmolken suggests. Right know, I believe @jpcima's data more than mine. In the illo I pasted I'm sort of "eyeballing" "oh, looks like the releases ends here". Not very reliable  for absolute numbers, esp when the definition is -85 db!

**Comment by kinwie** [comment]
Alright, I think Jpcima is correct. Just did a comparison again in sforzando. It is indeed 0.03

**Comment by DSmolken** [comment]
Confirmed, everybody seems to now agree that it's 0.03 in ARIA and 0.001 in rcg sfz and Cakewalk stuff. Sfizz will likely also update their default to this to match ARIA - will update ampeg_release again once they confirm that's done.
